packages/terra-application-name/examples/test-examples/ApplicationHeaderNameDefault.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Image from 'terra-image';
+import ApplicationHeaderName from '../../lib/ApplicationHeaderName';
+
+const color = { color: 'black' };
```

Comment:
IMO It would be better to set the background color to be different than white, than overriding the default font color.